server/server.go

```diff
@@ -1471,7 +1471,11 @@ func (s *Server) startMonitoring(secure bool) error {
 	s.mu.Unlock()
 
 	go func() {
-		srv.Serve(httpListener)
+		err := srv.Serve(httpListener)
+		if err != nil {
+			s.Errorf("Start %s monitor on %s error: %v", monitorProtocol,
```

Comment:
Should this be `s.Fatalf` to match other listeners like clients? 

Also maybe break out hp and make error consistent.

```go
	hp := net.JoinHostPort(opts.HTTPHost, strconv.Itoa(opts.HTTPPort))
	s.Fatalf("Error starting monitoring on port: %s, %q", hp, err)
```

```

Reply:
Actually, `hp` has been initialized above, thus we can reference it directly.

should the `err` be formatted with `%q` or `%v`?

Reply:
I usually use %v.